config/postcss-generate-classes/index.js

```diff
@@ -0,0 +1,54 @@
+const postcss = require('postcss');
+const parser = require('postcss-selector-parser');
+
+//TO DO: 
+//* figure out how to pass file name- to replace fd-button with component file name
+//* handle .#{$block}, [class*="#{$block}--"], a[class*="#{$block}--"], the [class*="#{$block}--"] selectors are getting picked up by the attribute selector block
+
```

Comment:
Additional to do: handle `[disabled]` attribute selector from fundamental-styles scss. Do not convert this to class.